addons/popochiu/engine/objects/clickable/popochiu_clickable.gd

```diff
@@ -72,7 +78,7 @@ func _ready():
 		$BaselineHelper.free()
 		$WalkToHelper.free()
 		
-		if get_node_or_null("InteractionPolygon"):
+		if not self is PopochiuCharacter and get_node_or_null("InteractionPolygon"):
```

Comment:
```suggestion
		if (
		        get_node_or_null("InteractionPolygon") # there is an InteractionPolygon node
		        and not self is PopochiuCharacter # the node is not a character
		):
```

I would style the if like that to make it visually similar to the one you changed above. This will make it simpler to spot the logic is inter-related.

Also, the comments "The node is not a character" is basically already understandable from the test itself. I would write a one-liner on why the not should or should not be a character or maybe reference the bug, like:

```gdscript
   # avoids unwanted reset of interaction polygon, see #<issue>
```

What do you think?

Reply:
I like it. I know an obvious comment wasn't necessary, but I couldn't think of anything else to add. Thanks for the suggestion.